### PR TITLE
Agent tours to separate file

### PR DESCRIPTION
**Pull request by atte-wsp**
Option to write agent data using separate file for tours. Separate person and trip tables is common way of storing travel survey data, so combining them in R is simple task.

+ More flexible analysis possibilities in R. Therefore no need to change Helmet model system, if mode specific extra analysis is needed (for example mode shares by gender).
- Consumes more hard disk space (from ~100 MB to 200 MB).

### CODE REVIEW

**Comment by atte-wsp** [comment]
This update is not necessary in our distribution analysis, but could be better in long term?

File: Scripts/modelsystem.py [hunk 1]
```diff
@@ -544,13 +544,23 @@ def _add_internal_demand(self, previous_iter_impedance, is_last_iteration):
...
+            fname0 = "agents"
+            fname1 = "tours"
+            # loop to first person that has tours
+            for person0 in self.dm.population:
+                if isinstance(person0.tours, list):
+                    break
+            # print person and tour attr to files
+            self.resultdata.print_line("\t".join(person0.attr), fname0)
+            self.resultdata.print_line(
+                "\t".join(person0.tours[0].attr), fname1)
```
Comment (zptro) [inline Scripts/modelsystem.py:556]: We can get the attribute names directly from `Person` and `Tour`:

````
from datatypes.person import Person
from datatypes.tour import Tour


self.resultdata.print_line("\t".join(Person.attr), fname0)
self.resultdata.print_line("\t".join(Tour.attr), fname1)
````

The loop is hence unnecessary (and probably not even working as intended, `person0.tours` is always a `list`, whether it is empty or not, the pythonical way to check if a `list` is non-empty is simply `if person0.tours`).

**Review by zptro — CHANGES_REQUESTED** [review]
I guess this change is alright. Just one small comment.

File: Scripts/modelsystem.py (hunk 1 above)
Comment (atte-wsp) [reply Scripts/modelsystem.py:556]: Thanks! I'm glad that there is a better solution as the one I implemented previously was quite complex way of getting headers for output files.

**Review by zptro — APPROVED** [review]

**Pull request merged**